src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -100,72 +100,68 @@ define(function (require, exports, module) {
-            });
-
-            var sectionClasses = classnames({
-                "libraries": true,
-                "section": true,
-                "section__sibling-collapsed": !this.props.visibleSibling
-            });
+                connected = libraryStore.getConnectionStatus();
 
             var containerContents;
-
             if (connected) {
+                var libraries = this.state.libraries,
+                    currentLibrary = libraries.get(this.state.selectedLibrary),
+                    selectedLayers = this.props.document.layers.selected;
+
                 containerContents = this.props.visible && !this.props.disabled && (
-                <div>
-                    <div className="formline">
-                        <LibraryList
-                            ref="libraryList"
-                            libraries={libraries}
-                            selected={currentLibrary}
-                            onLibraryChange={this._handleLibraryChange}
-                        />
-                        <SplitButtonList>
-                            <SplitButtonItem
-                                title={"CREATE LIBRARY"}
-                                className="button-plus"
-                                iconId="plus"
-                                onClick={this._handleLibraryAdd}
-                                />
-                            <SplitButtonItem
-                                title={"DELETE LIBRARY"}
-                                className="button-plus"
-                                iconId="libraries-delete"
-                                onClick={this._handleLibraryRemove}
-                                />
-                        </SplitButtonList>
+                    <div>
+                        <div className="formline">
+                            <LibraryList
+                                libraries={libraries}
+                                selected={currentLibrary}
+                                onLibraryChange={this._handleLibraryChange} />
+                            <SplitButtonList className="libraries__split-button-list">
+                                <SplitButtonItem
+                                    title={"CREATE LIBRARY"}
+                                    className="button-plus"
+                                    iconId="plus"
+                                    onClick={this._handleLibraryAdd} />
+                                <SplitButtonItem
+                                    title={"DELETE LIBRARY"}
+                                    className="button-plus"
+                                    iconId="libraries-delete"
+                                    onClick={this._handleLibraryRemove} />
+                            </SplitButtonList>
+                        </div>
+                        <Library addElement={this._handleAddElement} library={currentLibrary} />
```

Comment:
Would help readability if the props of Library were in their own lines like other components.